clustered/client/src/main/resources/ehcache-clustered-ext.xsd

```diff
@@ -82,13 +82,22 @@
   </xs:complexType>
 
   <xs:complexType name="connection-spec">
```

Comment:
We should really be enforcing one or the other of these in the XSD itself.

Reply:
Will do.